app/models/linkage_map.rb

```diff
@@ -11,5 +11,24 @@ class LinkageMap < ActiveRecord::Base
 
   has_many :map_locus_hits, foreign_key: 'linkage_map_id'
 
+  default_scope { includes(plant_population: :taxonomy_term) }
+
+  include Pluckable
+
+  def self.table_data(params = nil)
+    pluck_columns
+  end
+
+  def self.table_columns
```

Comment:
Out of curiosity - why these columns? Seems to be plenty more payload in `linkage_maps`.